html/pfappserver/lib/pfappserver/Base/Form.pm

```diff
@@ -171,6 +176,47 @@ sub id_validator {
    };   
 }
 
+before 'process' => sub {
+    my ($self) = @_;
+    foreach my $field (@{$self->fields}){
+        # Populate all the FingerbankSelect fields with the proper values extracted from the database.
+        if($field->type eq "FingerbankSelect"){
```

Comment:
I need to hook into the before/after of the form processing itself which I wasn't able to find in the formhandler field itself.

Let me know how you would do it and I'll make it work since I agree it would be better in the field itself